OpenCollarTeam/OpenCollar
# Clear menus to invalidate auth when public mode is turned off

Listening to LM_SETTING_EMPTY for "auth_public".
Uses RemoveMenuStride() to close menu listeners.

Should solve issue #1095

Basic inworld testing done and works so far.

HOW TO TEST:

Check that all normal collar menus still are working ok, with owner and a friend accessing the menus.

Set collar to public and let a co tester who is not owner try that RLV force sit works.
Then let the friend open a RLV/Forcesit meny and just keep it open.
Now change to non public, and after that the friend makes a force sit selection.
* This should now have no effect at all, and friend have to reopen menu

For comparison try the same on a basic 8.2.3 collar
